openhands/runtime/action_execution_server.py

```diff
@@ -199,6 +201,12 @@ async def run_ipython(self, action: IPythonRunCellAction) -> Observation:
 
             obs: IPythonRunCellObservation = await _jupyter_plugin.run(action)
             obs.content = obs.content.rstrip()
+            match = re.search(
+                r'<oh_aci_output>(.*?)</oh_aci_output>', obs.content, re.DOTALL
+            )
+            if match:
+                result_dict = json.loads(match.group(1))
```

Comment:
Just to confirm, this is not used yet, right?

Reply:
Going forward, it will be nice if we can import a pydantic class from openhands-aci, and convert this result dict to that class. But definitely not needed in this PR.

Reply:
No, it's already used because we already wrap them in the tag in `openhands-aci`'s main branch. I've tested manually and via 10 instances run but maybe we need to look a bit more closely here, just to make sure it won't break things.